demisto_sdk/commands/validate/validators/IN_validators/IN124_is_hiddenable_param.py

```diff
@@ -83,13 +77,37 @@ def get_invalid_hidden_params(
                         ParameterType.TEXT_AREA.value,
                         ParameterType.TEXT_AREA_ENCRYPTED.value,
                     )
-                    and self._is_replaced_by_type9(param.display or "", params)
+                    and self._is_replaced_by_type9(
+                        param.display or "", content_item.params
+                    )
                 )
+                or self.is_param_already_hidden(content_item, param)
             )
         ]
-        self.invalid_params[integration_name] = invalid_params
+        self.invalid_params[content_item.name] = invalid_params
         return invalid_params
 
+    def is_param_already_hidden(
+        self, content_item: Integration, param: Parameter
+    ) -> bool:
+        """
+        Return True if the param was already set to True in the old content object.
+        Args:
+            content_item (Integration): The integration to test.
+            param (Parameter): The current param to check.
+
+        Returns:
+            bool: True if the param was already set to True in the old content object. Otherwise, return False.
+        """
+        return bool(
+            (old_obj := content_item.old_base_content_object)
+            and content_item.old_base_content_object.git_status
+            in [GitStatuses.MODIFIED, GitStatuses.RENAMED]
```

Comment:
why arent these added to the head of the file like GItStatuses = []..?

Reply:
Good point tbh...